generators/chipyard/src/main/scala/TestSuites.scala

```diff
@@ -176,6 +95,7 @@ class TestSuiteHelper
         else            ((if (vm) rv32i else rv32pi), rv32u)
 
       addSuites(rvi.map(_("p")))
+      addSuites(rvu.map(_("p")))
       addSuites((if (vm) List("v") else List()).flatMap(env => rvu.map(_(env))))
```

Comment:
Is giving all the cores the same test-suite generator function actually the right thing to do here? Does this still produce the same set of test-suites for each core?

Reply:
I don't think we should generalize the tests that each core runs, some cores will be RV64GC, others may not so I think it is important for the cores to determine which set of tests to run.

Reply:
I have checked the code for Rocket, Boom and Ariane and it seems like they only have a few differences. Boom always executes `addSuites(env.map(rv64uf))` and also executes `addSuites(rvu.map(_("p")))` which no other core does. I tested Rocket and the extra suite `rvu` does not affect it, although I haven't test Boom yet. Maybe I can introduce another set of config to specify what testes they should use.